external/fv3fit/fv3fit/emulation/data/transforms.py

```diff
@@ -0,0 +1,115 @@
+"""
+Array transforms to build onto our batching pipeline into training.
+Generally assumes data to be in Sample x Feature shape.
+"""
+
+import logging
+import numpy as np
+import tensorflow as tf
+import xarray as xr
+from toolz.functoolz import curry
+from typing import Mapping, Sequence, Tuple, Union
+
+from vcm import get_fs, open_remote_nc
+
+
+logger = logging.getLogger(__name__)
+
+NumericContainer = Union[np.ndarray, xr.DataArray, tf.Tensor]
+ArrayDataset = Mapping[str, np.ndarray]
+TensorDataset = Mapping[str, tf.Tensor]
+AnyDataset = Mapping[str, NumericContainer]
+
+
+def open_netcdf_dataset(path: str) -> xr.Dataset:
```

Comment:
This I/O code is dissimilar to the other contents of this module. I suggest moving it someplace else.

Reply:
Moved to `fv3fit.emulation.data.io`